src/ContinuousEnergyNeutron.tex

```diff
@@ -109,29 +109,32 @@ \subsubsection{\textsf{ESZ} Block}\label{sec:ESZBlock}
   \end{tablenotes}
 \end{ThreePartTable}
 
-\subsubsection{\textsf{NU} Block}\label{sec:NUBlock}
-There are three possibilities for the \Block{NU}:
+\subsubsection{\textsf{NU} \& \textsf{DNU} Block}\label{sec:NUBlock}\label{sec:DNUBlock}
+
+The \Block{NU} is used to specify prompt and/or total \nubar and is present only if \jxs{2} > 0. Delayed \nubar data is specified in the \Block{DNU} (which is only present if \jxs{24} > 0) but it shares some of the tables defined in this section.
+
+When it is present, there are two possibilities for the \Block{NU}:
 \begin{enumerate}
-  \item {\bfseries\sffamily No NU Block.} (\jxs{2}=0)
-  \item {\bfseries\sffamily Either prompt or total \nubar\ is given (but not both).} (\jxs{2} > 0) \\
-    The \aceArray{NU} array begins at location \xss{KNU} where \var{KNU}=\jxs{2} + 1.
-  \item {\bfseries\sffamily Both prompt and total \nubar\ are given.} (\jxs{2} < 0).
+  \item {\bfseries\sffamily Either prompt or total \nubar\ is given (but not both).} (\xss{\jxs{2}} > 0) \\
+    A single \nubar array is given and it begins at location \xss{KNU} where {\sffamily \var{KNU} = \jxs{2}}.
+  \item {\bfseries\sffamily Both prompt and total \nubar\ are given.} (\xss{\jxs{2}} < 0).
+    Two \nubar arrays are given, one for prompt \nubar and another for total \nubar. The absolute value of \xss{\jxs{2}} is the location of the total \nubar array so that the locations for the two \nubar arrays are as follows:
```

Comment:
```suggestion
    Two \nubar\ arrays are given, one for prompt \nubar\ and another for total \nubar. The absolute value of \xss{\jxs{2}} is the location of the total \nubar\ array so that the locations for the two \nubar\ arrays are as follows:
```

Reply:
Should be changed now